src/common/Fields.js

```diff
@@ -45,6 +45,28 @@ export  class InputField extends React.Component {
   }
 }
 
+@withStyles(styles, {withTheme: true})
+export class CheckBox extends React.Component {
```

Comment:
Actually we should use material-ui checkboxes: https://material-ui.com/components/checkboxes/

Example use can be found in the registration form: https://github.com/dnd-al-krk/portal_frontend/blob/master/src/common/Register.js#L302